docs/system-design.md

```diff
@@ -51,14 +51,19 @@ Some units:
 
 - Token `{tok}`: A token that our protocol holds a balance of, mostly as backing for the RToken.
 
-A couple examples:
+Some examples:
 
 - In the USD+ RToken we have designed, the unit of account is USD. Among others, cUSDC is a collateral token with reference unit USDC and target unit USD, and aUSDP is a collateral token with reference token USDP and target unit USD.
 
 - Let's say we're building a pure-stable USD basket, out of USDC, USDP, and DAI. The unit of account would surely be USD. Each collateral token would also be its own reference unit, and its target would be USD.
 
+- Perhaps we're interested in a USD-denominated basket of blue-chip cryptocurrencies.  This type of rToken could be a 50/50 basket of wstETH and yvwBTC, where the reference units could be ETH and wBTC, respectively.  The target units would then be ETH & BTC, while the `{UoA}` would be USD.  Thus, the *value* of the rToken would flucuate (according to its unit-of-account), but all other necessary properties could be maintained.
+
 Separate from these, a number in dimension `{BU}` ("basket units") is an amount of current baskets.
 
+Regarding `{UoA}` and `{target}`:
+While it will usually be the case that a collateral's `{target}` will be the same as its RToken's `{UoA}`, this is by no means a requirement.  The `{UoA}` is a way to value the RToken in terms of a single unit, while each collateral's `{target}` is the expected value of its liability, or its `{ref}`.  As in example #3 above, an RToken's collaterals may have completely different `{target}` units, but be valued by the same `{UoA}`.
```

Comment:
```suggestion
### Regarding `{UoA}` and `{target}`

While it will usually be the case that a collateral's `{target}` will be the same as its RToken's `{UoA}`, this is by no means a requirement.  The `{UoA}` is a way to value the RToken and its collateral in terms of a single unit, while each collateral's `{target}` is the expected value of its liability, or its `{ref}`.  As in example #3 above, an RToken's collaterals may have completely different `{target}` units, but be valued by the same `{UoA}`.
```